client/src/screens/AnimeStatusScreen.tsx

```diff
@@ -37,6 +40,15 @@ const AnimeStatusScreen: FC = () => {
                     />
                 </Paper>
             </Modal>
+            <Modal
+                style={{ margin: "5%" }}
+                open={!!animeShown}
+                onClose={() => setAnimeShown(null as unknown as AnimeStatus)}
```

Comment:
this arrow function produces a new ref every render event better to use useCallback